task/openshift-client/0.2/README.md

```diff
@@ -0,0 +1,88 @@
+## OpenShift Client Task
+
+[OpenShift](http://www.openshift.com) is a Kubernetes distribution from Red Hat which provides `oc`, the [OpenShift CLI](https://docs.openshift.com/container-platform/4.1/cli_reference/getting-started-cli.html) that complements `kubectl` for simplifying deployment and configuration applications on OpenShift.
+
+Openshift-client runs commands against the cluster provided by the user via optional workspace and if not provided then it will take the cluster on which the `Task` is running.
+
+## Install the Task
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/openshift-client/0.2/openshift-client.yaml
+```
+
+## Parameters
+
+- **ARGS:** args to execute which are appended to `oc` e.g. `start-build myapp` (_default_: `help`)
+
+- **SCRIPT:** script of oc commands to execute e.g. `oc get pod $1 -0 yaml` This will take the first value of ARGS as pod name (_default_: `oc $@`)
+
+- **VERSION:** The OpenShift version to use (_default_: `4.7`)
+
+## Workspaces
+
+The `Task` uses 2 optional workspaces:
+
+- **manifest-dir**: The workspace which contains kubernetes manifests which we want to apply on the cluster.
+
+- **kubeconfig-dir**: The workspace which contains the the `kubeconfig` file if in case we want to run the `oc` command on another cluster.
```

Comment:
nit: 
```suggestion
- **kubeconfig-dir**: The workspace which contains the `kubeconfig` file if in case we want to run the `oc` command on another cluster.
```